qiskit_experiments/curve_analysis/standard_analysis/error_amplification_analysis.py

```diff
@@ -110,10 +110,16 @@ def _default_options(cls):
         """
         default_options = super()._default_options()
         default_options.result_parameters = ["d_theta"]
+        default_options.max_good_angle_error = np.pi / 2
+
+        return default_options
+
+    @classmethod
+    def _default_draw_options(cls):
+        default_options = super()._default_draw_options()
         default_options.xlabel = "Number of gates (n)"
         default_options.ylabel = "Population"
-        default_options.ylim = [0, 1.0]
-        default_options.max_good_angle_error = np.pi / 2
```

Comment:
Is `max_good_angle_error` an option that we no longer need?

Reply:
These are the draw options. It's moved to analysis options.

Reply:
Sorry this was a mistake. I move this back to the analysis option